extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1092,6 +1088,8 @@ export class AmpA4A extends AMP.BaseElement {
     const iframe = /** @type {!HTMLIFrameElement} */(
         createElementWithAttributes(
             /** @type {!Document} */(this.element.ownerDocument), 'iframe', {
+              height: this.creativeSize_.height,
```

Comment:
Can we add a comment that the height/width could be 'auto' which is ok as it will cause the creative to center?